recipes/hwloc/all/conanfile.py

```diff
@@ -0,0 +1,118 @@
+from conan import ConanFile
+from conan.tools.apple import is_apple_os, fix_apple_shared_install_name
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsDeps, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class HwlocConan(ConanFile):
+    name = "hwloc"
+    description = "Portable Hardware Locality (hwloc)"
+    topics = ("hwloc")
+    license = "BSD-3-Clause"
+    homepage = "https://www.open-mpi.org/projects/hwloc/"
+    url = "https://github.com/conan-io/conan-center-index"
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "disable_libxml2": [True, False]
```

Comment:
Options with default ``False`` tend to behave more intuitively than the opposite.
Given that the build system provides ``HWLOC_WITH_LIBXML2``, I would provide a ``with_libxml2`` option, with a default=``False``